python/CMakeLists.txt

```diff
@@ -1,16 +1,24 @@
 include(cmake/find_python_module.cmake)
 include(cmake/python_package_versions.cmake) # finds version
 
-FIND_PACKAGE(PythonInterp 2.7 EXACT)
+FIND_PACKAGE(PythonInterp)
 if (NOT DEFINED PYTHON_EXECUTABLE)
-   message("Python2.7 interpreter not found - Python wrappers not enabled")
+   message(WARNING "Python2 interpreter not found - Python wrappers not enabled")
+   set( BUILD_PYTHON OFF PARENT_SCOPE )
+   return()
+endif()
```

Comment:
I don't think  `Python2` makes sense.  We don't support Python 2.6, and there will never be a Python 2.8, hence Python 2.7 is the only real alternative.  It should say `"Python 2.7 interpreter not found"`.